### DISCUSSION TITLE
Remove Foreign Key Constraint

### DISCUSSION THREAD

**Discussion by yeanpei** [simolus3/drift#2772] [Q&A] [answered]
I would like to remove foreign key constraint for a table with data. Do I need to do any migration or can directly remove the reference from table column then it will works? If migration is needed, which function should I use?

**Comment by simolus3** [comment, marked as answer]
> Do I need to do any migration

Yes! But it's quite simple, you just need to call [this method](https://drift.simonbinder.eu/docs/migrations/api/#changing-column-constraints) in `onUpgrade` with the affected table.

**Reply by yeanpei** [reply]
Great! Thanks a lot!
